core/scss/utilities/mixins/typography/_intro-text.scss

```diff
@@ -1,16 +1,17 @@
 @charset "UTF-8";
 
 //
-// @font-lead
+// @intro-text
 //
 // For leading a page of text, summarizing a main point, and/or highlighting a
 // paragraph or quote.
 //
-// Style guide: Mixins.Typography.font-lead
+// Style guide: Mixins.Typography.intro-text
 //
-@mixin font-lead {
+@mixin intro-text {
   @include modular-typography(2);
```

Comment:
```suggestion
  @include modular-typography(2);
  @include font-smoothing;
```

Reply:
Combination of using font-smoothing and giving it regular font weight would make it look close to what we have in Figma.